@xen-orchestra/lite/src/views/pool/PoolNetworkView.vue

```diff
@@ -1,11 +1,200 @@
 <template>
-  <PageUnderConstruction />
+  <div class="pool-network-view">
+    <UiCard class="card">
+      <div class="networks-content">
```

Comment:
You can extract this section and the `host-internal-networks-content' section into child components, which will improve the readability and maintainability of this component.